demo/angular/e2e/src/app.e2e-spec.ts

```diff
@@ -10,7 +10,7 @@ describe('workspace-project App', () => {
 
   it('should display welcome message', async () => {
     await page.navigateTo();
-    expect(await page.getTitleText()).toEqual('porcupine-web-angular-demo app is running!');
+    expect(await page.getTitleText()).toEqual('porcupine-angular-demo app is running!');
```

Comment:
you can remove this whole `e2e` folder. It's for end-to-end testing that we don't use